pyensembl/transcript.py

```diff
@@ -400,7 +400,10 @@ def sequence(self):
         Spliced cDNA sequence of transcript
         (includes 5" UTR, coding sequence, and 3" UTR)
         """
-        return self.genome.transcript_sequences.get(self.id)
+        for transcript_sequences in self.genome.transcript_sequences:
+            if self.id in transcript_sequences:
+                return transcript_sequences.get(self.id)
```

Comment:
Hm, I agree that combining all transcripts into one dictionary may be a better solution. How then to store the pickled versions of the transcript FASTA files? Combine them into something like transcripts.pickle?